.devcontainer/welcome-message.txt

```diff
@@ -0,0 +1,7 @@
+👋 Welcome to "Practical Deep Learning for Coders" in Codespaces!
+
+🛠️  Your environment is fully setup with all the required softwares and machine learning libraries.
+
+🚀 To get started, either open the notebook file in the VS Code editor,
+   or open this Codespace with "Open in Jupyterlab" at https://github.com/codespaces
```

Comment:
This makes me think that the documentation should also include opening the notebooks in Jupyter directly from the `gh` cli

Reply:
Added!